storybook/src/components/autosuggest/autosuggest.core-stories.ts

```diff
@@ -74,6 +76,47 @@ const { Example, Minimal3Characters, InSearchbar } = autosuggestStories({
     };
 
     return {
+      autosuggestMarkTemplate: (
```

Comment:
Deze extra template is niet nodig, de basis kan `autosuggestSearchBarTemplate()` zijn maar met een extra `mark` functie.